backend/app/Http/Controllers/StudentController.php

```diff
@@ -66,4 +89,20 @@ public function logout()
 
     return response($response, 200);
   }
+
+  public function update(Request $request) {
+    $studentDetails = Auth::user();
+    $student = Student::find($studentDetails->id);
+
+    $student->name = $request->name;
+    $student->username = $request->username;
+    $student->email = $request->email;
+    $student->password = bcrypt($request->password);
+    $student->thumbnail = request()->file('thumbnail')->store('thumbnails');
+
+
+    $student->save();
```

Comment:
It is better to use eloquent in laravel, search about on this one because it is very useful
```php
$student->update([
   'name' =>  $request->name,
   'username ' =>  $request->username ,
// columns to be updated
]);
```

Reply:
Got it Sir, I'll work on it.

Reply:
No need `$student->save()` since you already use `$student->update()` (This one update database automatically)

Reply:
Noted Sir, I'll update this one.